drivers/docker/driver_test.go

```diff
@@ -1004,6 +1004,34 @@ func TestDockerDriver_SecurityOpt(t *testing.T) {
 	require.Exactly(t, cfg.SecurityOpt, container.HostConfig.SecurityOpt)
 }
 
+func TestDockerDriver_SecurityOptFromFile(t *testing.T) {
+	seccompPath := "./test-resources/docker/seccomp.json"
```

Comment:
I'd move this down closer to where it's used and keep the method preamble for test environment conditionals.